packages/twenty-front/.eslintrc.cjs

```diff
@@ -23,8 +25,10 @@ module.exports = {
       },
       plugins: ['project-structure'],
       settings: {
-        'project-structure/folder-structure-config-path':
-          'packages/twenty-front/folderStructure.json',
+        'project-structure/folder-structure-config-path':path.resolve(
```

Comment:
does this solve the issue !? If yes, this is GOLD

Reply:
(from root) `npx eslint ./packages/twenty-front -c ./packages/twenty-front/.eslintrc-ci.cjs` => works
(from packages/twenty-front)  `npx eslint . -c ./.eslintrc-ci.cjs` => also works

That's great! Learned something here!

Reply:
❤️